code/backend/batch/utilities/document_loading/Strategies.py

```diff
@@ -10,20 +14,12 @@ class LoadingStrategy(Enum):
 
 def get_document_loader(loader_strategy: str):
```

Comment:
I feel that this should really be a static `DocumentLoadingFactory.create(loader_strategy: str)` - but feel might be pushing it for this PR - (feel free to resolve!)